app/services/github/webhook_processors/add_new_repo.rb

```diff
@@ -18,12 +18,10 @@ def call
       private
 
       def create_configuration(repo_info)
-        configuration = ::DeploymentConfiguration.create!(
-          project: @project,
+        configuration = ::DeploymentConfiguration.find_or_create_by(project: @project, integration_type: ProjectsConstants::Providers::GITHUB, integration_id: repo_info.id)
+        configuration.update!(
```

Comment:
If it's important to update every time we've found configuration, no questions :)
if not:
```Ruby
find_or_create_by!(...) do |configuration|
  configuration.repo_path = 
  configuration.name =
  configuration.status =
end
```

Reply:
Yes, it's important